packages/amplify-graphql-relational-transformer/src/__tests__/__snapshots__/amplify-graphql-relations-with-custom-primary-key.test.ts.snap

```diff
@@ -659,20 +679,22 @@ exports[`Resolvers for custom primary key and relational directives should not g
 "#if( $ctx.stash.deniedField )
   #return($util.toJson(null))
 #end
-#if( $util.isNull($ctx.source.postId) )
+#set( $partitionKeyValue = $util.defaultIfNull($ctx.stash.connectionAttributes.get(\\"postId\\"), $ctx.source.postId) )
+#if( $util.isNull($partitionKeyValue) )
   #set( $result = {
   \\"items\\":   []
 } )
   #return($result)
 #else
   #set( $limit = $util.defaultIfNull($context.args.limit, 100) )
+  #set( $sortKeyValue0 = $util.defaultIfNull($ctx.stash.connectionAttibutes.get(\\"title\\"), $ctx.source.title) )
```

Comment:
Here, we are defining the sort key value, but not using it anywhere? Are we missing some step to set the sort key?

Reply:
This is expected as the CPK feature flag is not enabled. The sort key field will not be used in the query. This is kept for not breaking old projects.

Reply:
The test statement for this snapshot: `should not generate sort key field in implicit hasMany relation when CPK feature is disabled`